docs/source/api/v4/server_server_capabilities.rst

```diff
@@ -25,6 +25,7 @@ Gets all associations of :term:`Server Capabilities` to :term:`cache servers`.
 
 :Auth. Required: Yes
 :Roles Required: None
+:Permissions Required: SERVER-CAPABILITY-SERVER:READ, SERVER:READ, SERVER-CAPABILITY:READ
```

Comment:
idk if this needs a separate Permission set, these are really just a server field that got put in a separate endpoint for some reason.